debianpkg/backports/ubuntu12.04/debian/rules

```diff
@@ -135,18 +135,11 @@ override_dh_auto_configure:
 	fi
 
 override_dh_auto_build:
-	#dh_auto_build
-	$(MAKE)
-	dh_auto_build -- -C doc draft-zebra-00.txt
-
-
-	# doc/ is a bit crazy
 ifeq ($(GENERATE_PDF), 1)
-	dh_auto_build -- -C doc frr.pdf || true # pdfetex fails with exit code 1 but still produces a good looking .pdf
+	dh_auto_build -- -C doc user.pdf
```

Comment:
I believe this should be `FRR.pdf`  (no source for user.pdf)
`dh_auto_build -- -C doc FRR.pdf`